### ISSUE TITLE
[BUG] Settings are forgotten between updates

### ISSUE THREAD

**Issue by NominalWasTaken** [radj307/volume-control#98]
### Version

6.1.1-rev1

### Windows Version

Windows 11 Home Version 22H2 (OS Build 22621.2428)

### Actual Behaviour

Settings are forgotten after update. I had update once before and it reset my changes but I had hopped it was a one time thing but after the most recent update from 6.0.0 to 6.1.1-rev1 it seems after every update the settings are completey reset to default. Let me know if their is any more information I can provide to aid in solving this bug.

### Reproduction Steps

I dont know if im just updating wrong but I just 
1. shut the program down 
2. get the latest installer executable from the releases page
3. run the executable
4. Then when I open the application again it has reset

### Log

```shell
HH:mm:ss:fff             === Log Initialized @ Saturday, October 21, 2023 2:25:02 AM ===  { Filter: 30 (ALL_EXCEPT_DEBUG) }
20:25:02:456  [INFO]     The version number in the settings file was 0.0.0, settings will be upgraded to 6.1.1-rev1.
20:25:02:770  [INFO]     Volume Control version 6.1.1-rev1 (Installed)
20:25:02:817  [INFO]     Microsoft Windows 11 Home (64-bit)
20:25:03:354  [ERROR]    Failed to get the Process associated with session '' because of an exception:
                         {
                           'Message': 'Process with an Id of 1560 is not running.'
                           'HResult': '-2147024809'
                           'Source': 'System.Diagnostics.Process'
                           'TargetSite': {
                             'Name': 'GetProcessById'
                             'DeclaringType': 'System.Diagnostics.Process'
                             'Attributes': 'PrivateScope, Public, Static, HideBySig'
                             'CallingConvention': 'Standard'
                           }
                           'StackTrace': {
                             [0] at System.Diagnostics.Process.GetProcessById(Int32 processId, String machineName)
                             [1] at System.Diagnostics.Process.GetProcessById(Int32 processId)
                             [2] at VolumeControl.CoreAudio.AudioSession.GetProcess()
                           }
                         }
20:25:04:404  [CRITICAL] There was a problem with the hotkeys provided by the previous configuration. Attempting to load default hotkeys instead.
```

**Comment by Rajdave69** [comment]
Same error happens to me too.

**Comment by radj307** [comment]
Thanks for the report! I have identified the issue.  
Expect the fix in v6.2.0 shortly.

**Comment by radj307** [comment]
Fixed in 4f6b507838e8d68eab309b8b49633a8c2e5f87c6

The issue happened because the assembly attribute used to track the version number was changed and the code was not updated to match. I've added an exception for this case so the program will report the error and crash instead of silently failing.

**Comment by radj307** [comment]
Fix included in [v6.2.0](https://github.com/radj307/volume-control/releases/tag/6.2.0)
